src/Microsoft.AspNet.Server.Kestrel/Http/DateHeaderValueManager.cs

```diff
@@ -17,6 +18,9 @@ public class DateHeaderValueManager : IDisposable
         private readonly TimeSpan _timerInterval;
 
         private volatile string _dateValue;
+        private volatile int _activeDateBytes;
+        private byte[] _dateBytes0 = Encoding.ASCII.GetBytes("DDD, dd mmm yyyy hh:mm:ss GMT");
```

Comment:
Oops. I thought this was reallocating an array each time.

Might as well mark this a readonly.